add index page for docs (#14)
mirumee/saleor-sdk-python: docs/index.md

# Introduction

> Saleor SDK Python is a Python library which purpose is to contain code that is repeated in different Saleor applications.

Reviewer: ```suggestion
Saleor SDK Python is a Python library that implements tools and solutions useful in Saleor application development, including:

- the `saleor_sdk.crypto` package for working with Saleor's auth.
```

> You should read [Saleor's Documentation](https://docs.saleor.io/docs/3.x/developer/extending/overview) on the topic to get a grasp on the concepts of the "app framework" Saleor comes with.

Reviewer: ```suggestion
Before starting, you should familiarize yourself with Saleor's ["Extending Saleor"](https://docs.saleor.io/docs/3.x/developer/extending/overview) documentation and learn basic concepts like apps, webhooks and events.
```

> This SDK is agnostic of any framework, which makes it possible to use in all sorts of web (or otherwise) frameworks and even scripts.

Reviewer: ```suggestion
This SDK is framework agnostic. It can be used with any Python web framework, or without framework as part of a script.
```

> ## What about Python App Framework?

Reviewer: ```suggestion
## What about the Python App Framework?
```

> The old [saleor-app-framework-python](https://github.com/mirumee/saleor-app-framework-python) was very opinionated on how an application should be crated. It required FastAPI and in a specific version of it.  

Reviewer: ```suggestion
The [saleor-app-framework-python](https://github.com/mirumee/saleor-app-framework-python) has been deprecated and is no longer maintained.
```

## Installation

```
pip install saleor-sdk-python
```

> ## Key features

Reviewer: I would consider dropping this part and moving 2-3 most important things from this to "introduction".

Reviewer 2: Decided to keep it here.
However when we will introduce docs for marina then we will decide to move it to `introduction`

## Key features

> - CLI with tooling that helps to decode and encode Saleor IDs - this is useful when a entity type or ID needs to be recognized from a GraphQL ID

Reviewer: ```suggestion
- CLI tools automating common development tasks: testing, linting and publication.
- CLI utility for deserializing Saleor's GraphQL IDs values to reveal type names and their database IDs - helpful in debugging and local development.
- 
```
